dist/framebus.js

```diff
@@ -76,7 +86,10 @@
 
   function _packagePayload(event, args, origin) {
     var packaged = false;
-    var payload = { event: event };
+    var payload = {
+      event:  event,
```

Comment:
Incidentally, the `.eslintrc` enforces that all of the values line up vertically